package.json

```diff
@@ -156,11 +162,12 @@
     "typescript-eslint": "^7.18.0"
   },
   "dependencies": {
+    "@jimp/types": "^1.6.0",
```

Comment:
`jimp` already has it's own typescript declarations and has `@jimp/types` inside. No need to include extras that will be forgotten.

Remove this dependency and replace `JimpClass` with `JimpInstance` from `jimp`

Reply:
Actually not all places seem to work with `JimpInstance` :/